build.gradle

```
@@ -6,13 +6,15 @@ version '1.0-SNAPSHOT'
 
 repositories {
     mavenCentral()
+    maven { url 'https://jitpack.io' }
 }
 
 dependencies {
     testImplementation platform('org.junit:junit-bom:5.9.1')
     testImplementation platform('org.assertj:assertj-bom:3.25.1')
     testImplementation('org.junit.jupiter:junit-jupiter')
     testImplementation('org.assertj:assertj-core')
+    implementation 'com.github.woowacourse-projects:mission-utils:1.1.0'
```

```suggestion
    implementation ('com.github.woowacourse-projects:mission-utils:1.1.0')
```
이미 존재하던 implementation 방식과 통일시켜보는건 어떨까요?
사소하지만 첫 시작부터 사소한 것을 잘 지켜줘야 프로젝트가 커져도 통일성이 유지되더라구요 :)

---

이 부분까지 고려하지 못 했었네요!
build.gradle 파일도 형식 잘 맞춰주도록 하겠습니다👍